tests/Unit/TeamSpeakTest.php

```diff
@@ -100,30 +100,32 @@ public function testGroupQualification()
 
     public function testValidDisplayName()
     {
-        $account = $this->account;
         $validDisplayName = 'John Doe';
-        $this->assertEquals(true, $account->isValidDisplayName($validDisplayName));
+        $this->assertEquals(true, $this->account->isValidDisplayName($validDisplayName));
     }
 
     public function testInvalidDisplayName()
     {
-        $account = $this->account;
         $invalidDisplayName = 'John Do';
-        $this->assertEquals(false, $account->isValidDisplayName($invalidDisplayName));
+        $this->assertFalse($this->account->isValidDisplayName($invalidDisplayName));
     }
 
     public function testPartiallyValidDisplayName()
     {
-        $account = $this->account;
         $validDisplayName = 'John Doe - EGKK_TWR';
-        $this->assertEquals(true, $account->isPartiallyValidDisplayName($validDisplayName));
+        $this->assertTrue($this->account->isPartiallyValidDisplayName($validDisplayName));
     }
 
     public function testNotPartiallyValidDisplayName()
     {
-        $account = $this->account;
         $validDisplayName = 'John Do - EGKK_TWR';
-        $this->assertEquals(false, $account->isPartiallyValidDisplayName($validDisplayName));
+        $this->assertFalse($this->account->isPartiallyValidDisplayName($validDisplayName));
+    }
+
+    public function testAllowedDuplicates()
```

Comment:
How does this test allowed duplicates? As far as I can see, these would be allowed anyway.

Reply:
`John Do` vs `John Doe`. Checking that it doesn't except it just because there is a number at the end